client/dns_client_flow.ml

```diff
@@ -1,3 +1,11 @@
+
+let stdlib_random n =
+  let b = Cstruct.create n in
+  for i = 0 to pred n do
+    Cstruct.set_uint8 b i (Random.int 256)
+  done;
+  b
```

Comment:
could as well be required as a dependency from mirage-stdlib-random (but feels fine to have here imho)